test/parallel/test-os-process-priority.js

```diff
@@ -116,8 +116,10 @@ function checkPriority(pid, expected) {
     return;
   }
 
+  // On Windows setting PRIORITY_HIGHEST will only work for elevated user,
+  // for others it will be silently reduced to PRIORITY_HIGH
   if (expected < PRIORITY_HIGH)
-    assert.strictEqual(priority, PRIORITY_HIGHEST);
+    assert.ok(priority === PRIORITY_HIGHEST || priority === PRIORITY_HIGH);
```

Comment:
Instead of doing this for all cases, can we make an explicit check for Windows and possibly for elevated user status before making this kind of assertion?

Reply:
This is for Windows only (see line 114)

Reply:
I see, can we check for elevated status then? Perhaps using [this](https://superuser.com/a/1122031) method or something similar (without 3rd party utilities)?